nuxeo-core/nuxeo-core-mongodb/src/test/java/org/nuxeo/ecm/core/mongodb/seqgen/TestSequenceGeneratorWithMongoDB.java

```diff
@@ -64,17 +74,57 @@ public void testIncrement() throws Exception {
     }
 
     @Test
+    @LogCaptureFeature.FilterOn(logLevel = "WARN")
     public void testInitSequence() {
         UIDSequencer seq = uidGeneratorService.getSequencer();
+        assertTrue(seq.getClass().isAssignableFrom(MongoDBUIDSequencer.class));
 
-        seq.getNext("mySequence");
-        seq.getNext("mySequence");
+        seq.getNext("autoSequence");
+        seq.getNext("autoSequence");
+        assertTrue(seq.getNext("autoSequence") > 2);
 
         seq.initSequence("mySequence", 1);
         assertTrue(seq.getNext("mySequence") > 1);
+        assertTrue(seq.getNext("mySequence") < 10);
         seq.initSequence("mySequence", 10);
-        assertTrue(seq.getNext("mySequence") > 10);
+        assertTrue("Sequence should skip ahead to 10", seq.getNext("mySequence") > 10);
```

Comment:
What prevent us to use assertEquals(11, seq.getNext("mySequence")) ?

Reply:
Nothing!  I modified an existing test which uses `>` instead of `equals`.